### PR TITLE
Add xsi:type

### PR DESCRIPTION
**Pull request by opatrascoiu**
Some tests, especially the old ones (e.g. CL2 suite) do not specify the xsi:type for `<value>` tag in test cases. The new ones use them.

For consistency and to simplify the code for the runners, I've added the xsi:type for all thye test cases.

The files were pushed from Windows hence you have to ignore whitespaces when reviewing in GitHub.

### CODE REVIEW

**Comment by tarilabs** [comment]
Thanks for this!

We had this very exact same problem already in the past, in at least 2 occurrences which I recall, and action was taken to ensure files are modified properly --at least in the past it is NOT only a problem of "viewing" on github, it becomes a problem because git history get contaminated with non-necessary changes.

Like in the past occurrences, please allow me (or other volunteer) the time to rework the PR to avoid unnecessary changes, and keep the only really required changes.

**Comment by tarilabs** [comment]
@opatrascoiu please consider the PR raised on your repo, which should fix to preserve the line ending to avoid the issue mentioned above.
https://github.com/opatrascoiu/tck/pull/1/files?w=1

Raising a PR against your fork/repo was necessary as this PR did not enable maintainers to append commits.

Once that PR is merged on your fork/repo, the additional 2 commits will automatically reflect back into this PR page, and should avoid/fix the issue of the line ending.

Hope this helps!

p.s.: for the next time, as always, I strongly advise to allow maintainers to append commit to PR

**Comment by opatrascoiu** [comment]
Merged. Thanks.

**Comment by opatrascoiu** [comment]
We should probably add .gitattributes in this project to avoid differences between local settings.

https://help.github.com/en/articles/configuring-git-to-handle-line-endings

File: TestCases/compliance-level-3/0006-join/0006-join-test-01.xml [hunk 1]
```diff
@@ -18,57 +18,57 @@
...
 					</component>
 					<component name="name">
-						<value>Miller</value>
+						<value xsi:type="xsd:string">Miller</value>
 					</component>
 				</item>
 				<item>
 					<component name="id">
-						<value xsi:type="xsd:string">7976</value>
+						<value xsi:type="xsd:decimal">7976</value>
```
[the remark this answers is not in the document]
Comment (tarilabs) [reply TestCases/compliance-level-3/0006-join/0006-join-test-01.xml]: This was not "commit suggestion" yet.

File: TestCases/compliance-level-3/0006-join/0006-join-test-01.xml [hunk 2]
```diff
@@ -18,57 +18,57 @@
...
 					</component>
 					<component name="name">
-						<value>Ford</value>
+						<value xsi:type="xsd:string">Ford</value>
 					</component>
 				</item>
 				<item>
 					<component name="id">
-						<value xsi:type="xsd:string">7900</value>
+						<value xsi:type="xsd:decimal">7900</value>
```
[the remark this answers is not in the document]
Comment (tarilabs) [reply TestCases/compliance-level-3/0006-join/0006-join-test-01.xml]: This was not "commit suggestion" yet.

[the remark this answers is not in the document]
Comment (tarilabs) [reply TestCases/compliance-level-3/0006-join/0006-join-test-01.xml]: This specific line of code change suggestion is not resolved, as it is not "commit suggestion", yet.

**Pull request merged**